.palantir/revapi.yml

```diff
@@ -1025,14 +1025,11 @@ acceptedBreaks:
       new: "class org.apache.iceberg.types.Types.NestedField"
       justification: "new Constructor added"
     org.apache.iceberg:iceberg-core:
-    - code: "java.method.visibilityReduced"
```

Comment:
for other reviewers: I've checked and these blocked have been moved when running 
```
./gradlew :iceberg-core:revapiAcceptBreak --justification "Serialization across versions is not supported"  \
          --code "java.class.defaultSerializationChanged" \
          --old "class org.apache.iceberg.GenericManifestFile" \
          --new "class org.apache.iceberg.GenericManifestFile"
```